### PR TITLE
Essential read (Added type mixins)

### PR DESCRIPTION
**Pull request by zeftilldeath**
Atom seems to have done a lot of tidying too. Thanks @sndrs

### CODE REVIEW

File: static/src/stylesheets/module/onward/_essential-read.scss [hunk 1]
```diff
@@ -3,14 +3,11 @@
...
-        line-height: 2.5rem;
-        font-weight: normal;
-	    
+        @include font($f-serif-headline, 400, 36px);
+
         @include mq($until: mobileLandscape) {
             float: initial;
-            font-size: 1.75rem;
-            line-height: 2rem;
+            @include font($f-serif-headline, 400, 28px);
```
Comment (sndrs) [inline static/src/stylesheets/module/onward/_essential-read.scss]: isn't this `@include fs-headline(4);`?

File: static/src/stylesheets/module/onward/_essential-read.scss [hunk 2]
```diff
@@ -3,14 +3,11 @@
     margin-bottom: $gs-baseline;
 
     .fc-container__header__title {
-        font-size: 2.25rem;
-        line-height: 2.5rem;
-        font-weight: normal;
-	    
+        @include font($f-serif-headline, 400, 36px);
```
Comment (sndrs) [inline static/src/stylesheets/module/onward/_essential-read.scss]: `@include fs-headline(6);`?

**Comment by sndrs** [comment]
not sure if you've seen this https://github.com/guardian/guss-typography, but it has presets for type. it's good to stick to them if possible

**Comment by zeftilldeath** [comment]
@sndrs Even better. Thanks for that. My resources bookmarks folder is getting bigger and bigger.

**Comment by johnduffell** [comment]
lots of (intended?) whitespace changes - easier to view the diff with parameter [?w=0](https://github.com/guardian/frontend/pull/11209/files?w=0)

**Pull request closed without merging**